pkg/controller/utils/errorscrub.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"regexp"
+)
+
+var (
+	awsRequestIDRE = regexp.MustCompile(`(, )*(request id|Request ID): ([-0-9a-f]+)`)
```

Comment:
This is very AWS specific regex but I have still used it for GCP and Azure actuators as well just to be safe

Reply:
Could use case-insensitive here:
```suggestion
	awsRequestIDRE = regexp.MustCompile(`(, )*(?i:request id): ([-0-9a-f]+)`)
```
Also, I'm curious why you're matching & scrubbing the `, `s.

Reply:
I wonder if, rather than chopping this whole chunk out of the string, you might prefer to simply replace the ID itself with a static string.

```go
	myre := regexp.MustCompile("(?i)(request id: )(?:[-[:xdigit:]]+)")
	fmt.Println(myre.ReplaceAllString("foo, Request id: abc-123, bar", "${1}XXXX"))
```
Result: `foo, Request id: XXXX, bar`

Reply:
Just a message to add to the reason (or logs in the future?) so no need to display the request id IMO